src/gtest/test_history.cpp

```diff
@@ -43,10 +43,14 @@ class FakeCoinsViewDB : public CCoinsView {
                     const uint256 &hashBlock,
                     const uint256 &hashSproutAnchor,
                     const uint256 &hashSaplingAnchor,
+                    const uint256 &hashOrchardAnchor,
                     CAnchorsSproutMap &mapSproutAnchors,
                     CAnchorsSaplingMap &mapSaplingAnchors,
+                    CAnchorsOrchardMap &mapOrchardAnchors,
                     CNullifiersMap &mapSproutNullifiers,
-                    CNullifiersMap saplingNullifiersMap) {
+                    CNullifiersMap &mapSaplingNullifiers,
+                    CNullifiersMap &mapOrchardNullifiers,
+                    CHistoryCacheMap &historyCacheMap) {
```

Comment:
This is so frustrating. This is why I really want to get https://github.com/zcash/zcash/pull/5244 merged.

Reply:
Now that that’s merged, this convo is resolved 😆